arviz/plots/backends/bokeh/distplot.py

```diff
@@ -119,8 +133,42 @@ def _histplot_bokeh_op(values, values2, rotated, ax, hist_kwargs):
     if hist_kwargs.pop("cumulative", False):
         hist = np.cumsum(hist)
         hist /= hist[-1]
-    if rotated:
-        ax.quad(top=edges[:-1], bottom=edges[1:], left=0, right=hist, **hist_kwargs)
+
+    if is_circular:
+
+        if is_circular == "degrees":
+            edges = np.deg2rad(edges)
+            labels = ["0°", "45°", "90°", "135°", "180°", "225°", "270°", "315°"]
```

Comment:
We are always showing a full circle plot?

Reply:
I think that's what everybody does (I know this is not a very good argument, haha)